c++/src/kj/common.h

```diff
@@ -1932,6 +1932,17 @@ class ArrayPtr: public DisallowConstCopyIfNotConst<T> {
   // Syntax sugar for invoking U::from.
   // Used to chain conversion calls rather than wrap with function.
 
+  inline void fill(T t) { 
+    // Fill the area by copying t over every element.
+
+    if constexpr (sizeof(T) == 1) {
+      // use memset for small T types, needs to be convertible to kj::byte
+      memset(begin(), (byte)t, size());
+    } else {
+      for (size_t i = 0; i < size_; i++) { ptr[i] = t; } 
```

Comment:
super minor nit: `std::fill(...)` would work here too, right? Any reason not to use it?

Reply:
@kentonv is usually against using `std::`, happy to change if I'm wrong.

Reply:
> How often do we actually fill an array with anything other that zeros?

There are quite a few usages in tests (which deserve this method too imo?). Outside of tests there's usage with ' ' in stringify.c++.